src/lib/components/Map/Map.js

```diff
@@ -622,7 +622,9 @@ class Map extends Component {
           (
             <div id={this.props.mapId} style={{
               width: mapWidth,
-              display: this.props.MAP.blockLoad ? 'none' : 'inline'
+              display: this.props.MAP.blockLoad
+                || (this.props.VIEW && !this.props.VIEW.showMap)
+                ? 'none' : 'inline'
```

Comment:
Ah yeah the `VIEW` reducer, I had to add `showMap` to the `defaultState` in service-mapping and this worked like a charm!